dolphinscheduler-alert/dolphinscheduler-alert-server/src/main/java/org/apache/dolphinscheduler/alert/AlertServer.java

```diff
@@ -88,24 +91,23 @@ public void close() {
     public void destroy(String cause) {
 
         try {
+            // set stop signal is true
             // execute only once
-            if (Stopper.isStopped()) {
+            if (!Stopper.stop()) {
+                logger.warn("AlterServer is already stopped");
```

Comment:
If `!Stopper.stop()` why is it already stopped? Shouldn't `!Stopper.stop()` means it's not stopped yet?